### PR TITLE
Fix popup display while zoomed out

### PR DESCRIPTION
**Pull request by VijayanB**
### Description
While zoom out, maps will have multiple copies of world map. Before this fix, popup will be displayed always at first copy even if users are hovering over another copy. This will make sure that hover shows up on where pointer is hovered.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/tooltip/create_tooltip.tsx [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 
 export function isTooltipEnabledLayer(
   layer: MapLayerSpecification
 ): layer is DocumentLayerSpecification {
   return (
     layer.type !== 'opensearch_vector_tile_map' &&
     layer.type !== 'custom_map' &&
-    layer.source.showTooltips === true
+    layer.source.showTooltips === true &&
+    !!layer.source.tooltipFields?.length
```
Comment (naveentatikonda) [inline public/components/tooltip/create_tooltip.tsx:23]: Will the last condition `!!layer.source.tooltipFields?.length` returns false if tooltipFields are not provided after enabling toolTips?

Comment (VijayanB) [reply public/components/tooltip/create_tooltip.tsx:23]: showTooltips is a flag where user controls whether to show toolTip or not, however, tooltipFields contains list of tooltip fields. The last condition checks whether length of tooltipFields is > 0 or not

Comment (naveentatikonda) [reply public/components/tooltip/create_tooltip.tsx:23]: Yes, I got it. But, I'm trying to understand what will be the value of `!!layer.source.tooltipFields?.length` if no tooltipFields are provided. If it is not an optional field it's value will be false. But, what will be the value when it is not optional?

Comment (VijayanB) [reply public/components/tooltip/create_tooltip.tsx:23]: false

**Review by naveentatikonda — DISMISSED** [review]

**Review by naveentatikonda — APPROVED** [review]

**Review by junqiu-lei — APPROVED** [review]

**Pull request merged**